src/NServiceBus.Persistence.CosmosDB.PessimisticLock.AcceptanceTests/SetupFixture.cs

```diff
@@ -51,14 +49,17 @@ public async Task OneTimeTearDown()
             CosmosDbClient.Dispose();
         }
 
-        static string GetEnvironmentVariable(string variable, string fallbackEmulatorConnectionString)
+        public static string GetConnectionStringOrFallback(string environmentVariableName = "CosmosDBPersistence_ConnectionString", string fallbackEmulatorConnectionString = EmulatorConnectionString)
         {
-            var candidate = Environment.GetEnvironmentVariable(variable, EnvironmentVariableTarget.User);
-            var environmentVariableConnectionString = string.IsNullOrWhiteSpace(candidate) ? Environment.GetEnvironmentVariable(variable) : candidate;
+            var candidate = Environment.GetEnvironmentVariable(environmentVariableName, EnvironmentVariableTarget.User);
+            var environmentVariableConnectionString = string.IsNullOrWhiteSpace(candidate) ? Environment.GetEnvironmentVariable(environmentVariableName) : candidate;
 
             return string.IsNullOrEmpty(environmentVariableConnectionString) ? fallbackEmulatorConnectionString : environmentVariableConnectionString;
         }
 
+        public static bool IsRunningWithEmulator => GetConnectionStringOrFallback() == EmulatorConnectionString;
+
+        const string EmulatorConnectionString = "AccountEndpoint = https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
```

Comment:
can we centralise this so that if it needs changing it only needs to be changed in one place?

Reply:
Followed through on another PR https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/726